website/docs/clustering.md

```diff
@@ -165,7 +165,8 @@ With the release of Hudi version 0.9.0, we can schedule as well as execute clust
 specify the `—mode` or `-m` option. There are three modes:
 
 1. `schedule`: Make a clustering plan. This gives an instant which can be passed in execute mode.
-2. `execute`: Execute a clustering plan at given instant which means --instant-time is required here.
+2. `execute`: Execute a clustering plan at a given instant which means --instant-time is required. 
```

Comment:
@nsivabalan are the docs forward-looking, or do they have to capture nuances of all versions as well?  I'm wondering if docs are only looking forward, we don't have to mention that `--instant-time is required` for older versions.

Reply:
Good call, since docs are versioned, the doc needs to describe current state, those that want to understand previous versions go to old versions of docs. I updated, thanks

Reply:
yes. we have versioned docs. so should be ok.